src/main/java/ui/services/PBHealthService.java

```diff
@@ -14,34 +14,18 @@
 import taiga.model.query.sprint.UserStoryDetail;
 import taiga.util.pbAnalysis.PBHealthHelper;
 
-/**
- * PBHealth class is responsible for calculating and visualizing the health of the Product Backlog.
- * The health is determined by analyzing the distribution of User Stories across different states
- * (New, In Progress, Done, and Testing) within Sprints. This analysis helps in understanding
- * the readiness of the backlog for future sprints, identifying bottlenecks, and ensuring a balanced
- * workload throughout the development process.
- * 
- * Calculation Model:
- * - Health Score = (WeightedSum(Done) + WeightedSum(InProgress) + WeightedSum(New)) / TotalUserStories
- *   - Where WeightedSum(State) = Sum(UserStoriesInState * StateWeight)
- *   - StateWeights reflect the priority of having stories in certain states, e.g., Done > InProgress > New
- * - Additionally, the age of User Stories in each state can be analyzed to identify stale items.
- *
- * Parameters:
- * - Sprint: The current Sprint context to analyze User Stories' distribution.
- * - UserStoriesInNew: ObservableList tracking User Stories in the 'New' state.
- * - UserStoriesInProgress: ObservableList tracking User Stories in 'In Progress'.
- * - UserStoriesInDone: ObservableList tracking User Stories marked as 'Done'.
- * - UserStoriesInTesting: Tracks User Stories in 'Testing' if applicable.
- */
 public class PBHealthService extends Service<Object> {
     private Sprint sprint;
     private SimpleObjectProperty<Double> pbHealthRatio;
```

Comment:
Could be simplified to `DoubleProperty` and instantiated with `SimpleDoubleProperty`

Reply:
Done